.gitignore

```diff
@@ -6,3 +6,12 @@ __pycache__/
 
 # for installing Forest in editable mode when developing
 forest.egg-info/
+
+#We don't want to add data to github
+tutorials/data
+tutorials/comm_output
+tutorials/gps_output
+tutorials/gps_summary.csv
+
+#Remove checkpoints data because we don't need to double-track files
+tutorials/.ipynb_checkpoints
```

Comment:
This is a directory, so should have a trailing slash: `tutorials/.ipynb_checkpoints/`
https://www.atlassian.com/git/tutorials/saving-changes/gitignore#git-ignore-patterns

Reply:
Actually, to be precise it should be `/tutorials/.ipynb_checkpoints/` for this directory. The same applies for the lines above.